### PR TITLE
Update chaos-bugbounty-list.json

### PR DESCRIPTION
**Pull request by pussycat0x**
added new programs PlayStation, EC-council

### CODE REVIEW

File: chaos-bugbounty-list.json [hunk 1]
```diff
@@ -1752,6 +1752,26 @@
...
+            "playstation.net",
+            "sonyentertainmentnetwork.com",
+            "api.playstation.com",
+            "playstation.com"
+         ]
+      }
+      ,
+      {
+         "name": "EC-Council",
+         "url": "https://www.eccouncil.org/",
```
Comment (bauthard) [inline chaos-bugbounty-list.json:1770]: Can you please update the URL with https://www.eccouncil.org/bug-bounty/

File: chaos-bugbounty-list.json [hunk 2]
```diff
@@ -1752,6 +1752,26 @@
...
          ]
+      },
+      {
+         "name": "Playstation",
+         "url": "https://hackerone.com/playstation",
+         "bounty": true,
+         "domains":[
+            "playstation.net",
+            "sonyentertainmentnetwork.com",
+            "api.playstation.com",
```
Comment (bauthard) [inline chaos-bugbounty-list.json:1763]: Please remove `api.playstation.com` as it only requires domain name as input which is already in the list.

**Comment by bauthard** [comment]
nvm, I can handle this on my side.

File: chaos-bugbounty-list.json (hunk 1 above)
Comment (pussycat0x) [reply chaos-bugbounty-list.json:1770]: done

File: chaos-bugbounty-list.json (hunk 2 above)
Comment (pussycat0x) [reply chaos-bugbounty-list.json:1763]: done

**Pull request merged**